### ISSUE TITLE
The documentation does not allow building.

### ISSUE THREAD

**Issue by heretic13** [EOSIO/eos#2867]
On the page https://github.com/EOSIO/eos/wiki/Local-Environment#building-eosio there is a link "Manually Building EOSIO", which refers to nowhere.

There is no other way to build eos except to run the automatic installation script **eosio_build.sh**

The automatic installation script does not allow building eos - https://github.com/EOSIO/eos/issues/2861 (it is closed unilaterally by the developer, without trying to solve the problem).

Bottom line: It's impossible to build an eos from scratch. The documentation is incorrect (or incomplete). The official way of building contains errors. The developer ignores the problem.
